File: 13.block循环引用/13.block循环引用/ViewController.m
//
//  ViewController.m
//  13.block循环引用
//
//  Created by 浅爱 on 16/3/6.
//  Copyright © 2016年 y. All rights reserved.
//

#import "ViewController.h"
#import "NetworkTools.h"

@interface ViewController ()

@property (nonatomic, strong) NetworkTools *tools;

@end

@implementation ViewController

- (void)viewDidLoad {
    
    [super viewDidLoad];

    self.tools = [[NetworkTools alloc] init];
    
    
    [self.tools loadData:^(NSString *result) {
        
        NSLog(@"%@", result);
        
        NSLog(@"%@", self.view);
        
    }];
    
}


-(void)dealloc {

    NSLog(@"ViewController dealloc");


}

@end

File: 03.常量和变量/03.常量和变量/ViewController.swift
//
//  ViewController.swift
//  03.常量和变量
//
//  Created by 浅爱 on 16/3/5.
//  Copyright © 2016年 y. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        demo1()
        
    }

    
    func demo1() {
    
        let a = 10
        
        var b = 20
        
        b = 30
        
        print(b)
        
        // 显示指导类型
        let c: Double = 7.727181034890
        
        let result = Double(a) + c
        
        print(result)
    
    }
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

